website/content/docs/agent-and-proxy/agent/template.mdx

```diff
@@ -112,8 +112,11 @@ failures.
   includes connections in the dialing, active, and idle states.
 
 - `lease_renewal_threshold` `(float: 0.9)` - How long Vault Agent's template
-  engine should wait for to refresh dynamic, non-renewable leases, measured as
-  a fraction of the lease duration.
+  engine should wait to refresh dynamic, non-renewable leases, measured as
+  a fraction of the lease duration. For certificates rendered using the `pkiCert`
+  template function without leases, the `Not After` attribute is treated as the
+  lease end time, and the certificate will rotate at 70% of the difference between
```

Comment:
> and the certificate will rotate at 70%

Is the 70% related to this configuration at all? For example, by default, this is set to 0.9, would it be 90% by default?